sceptre/cli/template.py

```diff
@@ -137,3 +138,33 @@ def fetch_remote_template_command(ctx, path):
             output.append(template)
 
     write(output, context.output_format)
+
+
+@click.command(name="detect-stack-drift", short_help="Detects stack drift on running stacks.")
```

Comment:
This function still is on the `template` module. But I don't think it belongs here. See my comments on this PR at the bottom where I'm requesting changes.